### ISSUE TITLE
Not possible to skip SD-Card backup

### ISSUE THREAD

**Issue by micressor** [digitalbitbox/bitbox-wallet-app#1694]
If there is a new BitBox02, there is no way to skip sd card backup.

The app forces a backup to the SD card.

However, if I prefer to make a paper backup or steal wallet backup, I can't currently do that.

 Safe for beginners but not ideal for experienced bitboxers.

You could at least with warnings or a switch
`/opt/bitbox/BitBox --do-not-force-sd-card-backup`
make this possible for experienced bitboxers. ;-)

Found in version: 4.32.0

**Comment by micressor** [comment]
> Can you elaborate why exactly you prefer to skip the microSD card backup?

When the bitbox02 forces me to write to the SD card, I am confronted with erasing the SD card again safely, because the backup the sd-card is un-encrypted. If it's encrypted anyway, it doesn't matter so much, even if I don't need the SD card any further then.

**Comment by benma** [comment]
Your 24-words backup is also not encrypted. You could store the microSD card along with your 24-word backup. It makes restoring a lot quicker and easier.

**Comment by micressor** [comment]
@benma Confirmed. Restoring is easier. The app should just make sure that either an SD card backup was made or the words were properly written down on paper (by asking for the 24 words).

I would still appreciate it if the BitBoxApp didn't force you to do an SD card backup.

**Comment by benma** [comment]
This feature will be available in the upcoming firmware release [v9.13.0](https://github.com/digitalbitbox/bitbox02-firmware/blob/master/CHANGELOG.md).

We are still pondering if and how to add this option to the BitBoxApp itself.

**Comment by benma** [comment]
We plan on enabling this in the BitBoxApp. Assigning @thisconnect as he is currently working on it here:

https://github.com/digitalbitbox/bitbox-wallet-app/pull/2102

**Comment by NicolaLS** [comment]
is this closed by #2137 ?

**Comment by sjentzsch** [comment]
Didn't test or verify yet, but according to the latest release notes this should now be possible, yes:
https://github.com/digitalbitbox/bitbox-wallet-app/releases/tag/v4.38.0

> Add advanced setup with skip microSD card or 12-word seed options

@thisconnect guess we can close this one?

**Comment by thisconnect** [comment]
yes indeed, it is now possible to create a new wallet using recovery words instead of microSD card backup by using "advanced setup options" in v4.38.0, see https://bitbox.swiss/blog/bitbox-06-2023-bellinzona-update/

![Screen_Shot_2023-06-07_at_20 01 40](https://github.com/digitalbitbox/bitbox-wallet-app/assets/546900/27637943-d8a0-457d-8fc1-96f97a4ebf64)

**Comment by satoshinotdead** [comment]
I appreciate this! And I have a big question:

Initializing a new wallet (on a previously SD backed) with the same words should broke the SD backup or keep it working as before?

**Comment by benma** [comment]
Could you rephrase that question? I am not sure I understand your question.

**Comment by satoshinotdead** [comment]
@benma of course, sorry if I didn't explain myself correctly.

I lost my unencrypted SD card backup, but I still have my recovery phrase. If I restart my wallet and restore it using the recovery phrase, will the unencrypted SD card backup that BitBox generated without my consent still function as a backup?

Do I need to move my funds and incur fees, or can I simply restore my wallet resetting the wallet to effectively invalidate the unwanted, non-encrypted backup?

**Comment by benma** [comment]
The existing sdcard carries the same information as your recovery phrase, and anyone with access to it has access to your funds. If there is a chance that someone could have access to the sdcard, then it's best to create a new wallet/seed and move the funds.
